events/demolition_derby.py

```diff
@@ -0,0 +1,212 @@
+"""
+Bot makers:
+
+To support this event type, you must listen on matchcomms for an event shaped like this (see DerbySpecification):
+{
+  "event_type": "DemolitionDerby",
+  "perma_death": true,
+  "max_duration": 60.0,
+  "starts": [
```

Comment:
If I'm making a bot, should I ignore this and just look at the game tick packet during the countdown to see where everybody has spawned? From the message alone, I wouldn't know which index in "starts" was me.

Reply:
The index in the array should correspond with the index in the packet, but yeah it's not guaranteed. Maybe we can remove the start physics from the message. In waypoint race you can precompute your path or something, but here it's not very useful